### PR TITLE
Moved one-off scss into Panel.scss

### PR DESCRIPTION
**Pull request by patfire12345**

### CODE REVIEW

File: client/src/components/Panel/Panel.scss [hunk 1]
```diff
@@ -108,3 +108,12 @@ button.panel-heading-utils {
   margin: 20px 60px 20px 60px;
   border-top: 1px solid rgba($separatorColor, 0.5);
 }
+
+.panel-source-link {
+  font-weight: bold;
```
Comment (Stephen-ONeil) [inline client/src/components/Panel/Panel.scss:113]: Should use the centrally defined variables when setting font weights, `$heavyFontWeight` etc.

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]
Since both classes only set bold, I'd recommend deleting them and just adding the `bold` class to both.

File: client/src/components/Panel/Panel.scss (hunk 1 above)
Comment (patfire12345) [reply client/src/components/Panel/Panel.scss]: Done!

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]
To clarify, there's a bootstrap utility class named `bold` (we override it with our scss variable for heavy font weight somewhere). I was suggesting to just use that.

File: client/src/components/Panel/Panel.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @import "src/common_css/common-variables";
+@import "src/extended_bootstrap_css/bootstrap-wet-fixes-extensions";
```
Comment (Stephen-ONeil) [inline client/src/components/Panel/Panel.scss:2]: Not necessary. The `extended_bootstrap_css` is loaded globally early in the sites startup.

Not recommended either. If a JS module needed to guarantee multiple specific style sheets were loaded we'd have the imports inside the JS. That way webpack can set up common chunks and avoid downloading the styles multiple times. If you `@import` the styles directly in the stylesheet, the output CSS will have those styles inlined (and webpack won't know, so duplication will likely result). 

The main use of `@import` in our code base would be importing SCSS utilities like the variables declared in `common-variables` or SCSS utilities from the `Math` name space, etc. Those sorts of things can be thought of as being transformed away in the output CSS, just being replaced with the equivalent value.

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]

File: client/src/components/Panel/Panel.scss (hunk 2 above)
Comment (patfire12345) [reply client/src/components/Panel/Panel.scss]: Thanks! Removed the import now.

**Review by Stephen-ONeil — APPROVED** [review]
:+1:

**Pull request merged**